hermes-common/src/main/java/pl/allegro/tech/hermes/common/broker/BrokerDetails.java

```diff
@@ -5,11 +5,31 @@
     private String host;
     private int port;
 
+    private int unusedStuff;
+    private int zero = 0;
+
     public BrokerDetails(String host, int port) {
         this.host = host;
         this.port = port;
     }
 
+    private void unusedMessedUpMethod() {
+        label:
+        while (true) {
+            try {
+                if (zero == 1337) {
+                    System.out.println("look ma, zero == 1337");
+                    throw new Exception("throwing exception");
+                } else {
+                    zero += 1;
+                    break label;
+                }
+            } catch (Throwable e) {
```

Comment:
\[PMD\] WARNING: A catch statement should never catch throwable since it includes errors\.